p2p/simulations/http_test.go

```diff
@@ -234,10 +234,11 @@ func (t *testService) Snapshot() ([]byte, error) {
 // * get and increment a counter
 // * subscribe to counter increment events
 type TestAPI struct {
-	state     *atomic.Value
-	peerCount *int64
-	counter   int64
-	feed      event.Feed
+	state               *atomic.Value
+	peerCount           *int64
+	counter             int64
+	activeSubscriptions int64
```

Comment:
I think you want `sync.WaitGroup` here. Also maybe just `subCount`, no need to be overly verbose.